unity-renderer/Assets/Rendering/Shaders/Toon/Compiled/DCLSimpleLighting.hlsl

```diff
@@ -32,11 +32,13 @@ half4 DCL_SimpleFragmentPBR(InputData inputData, SurfaceData surfaceData)
 #endif
 
     #if defined(_SCREEN_SPACE_OCCLUSION)
+    #if !defined(_SSAO_OFF)
```

Comment:
```suggestion
    #if !defined(_SSAO_ON)
```
Can please invert the if to avoid double negatives? It makes code hard to maintain.

Reply:
Sure, I followed the other directives such as  `_RECEIVE_SHADOWS_OFF`. I also think it's way less problematic as it is so by default the SSAO is enabled (because the materials does not contains the `_SSAO_OFF` keyword).